index.js

```diff
@@ -0,0 +1,4 @@
+import CreateTodo from "./components/CreateTodo.js";
+
+CreateTodo();
```

Comment:
As this is the entry point of our app, let's name it more appropriately.

```suggestion
InitializeTodoApp();
```